src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer/index.jelly

```diff
@@ -113,35 +113,35 @@
                         <f:textbox name="gerritBuildStartedVerifiedValue"
                                    value="${it.config.gerritBuildStartedVerifiedValue}"
                                    default="${com.sonyericsson.gerrithudsontrigger.config.Config.DEFAULT_GERRIT_BUILD_STARTED_VERIFIED_VALUE}"
```

Comment:
Yes the textbox defaults. You should maybe also look over the defaults that are set when a new server config is created.